### ISSUE TITLE
evicted helm-operator pods clutter cluster

### ISSUE THREAD

**Issue by erkannt** [fluxcd/helm-operator#449]
**Describe the bug**

`helm-operator` keeps getting evicted due to node filesystem pressure

**To Reproduce**

Steps to reproduce the behaviour:
1. run helm-operator on node with small ephemeral-storage
2. pull lots of/big images?

**Expected behavior**

Any of:
- helm-operator purges old images periodically
- user can set limit/request via chart
- helm-operator somehow handles eviction to avoid having evicted pods pile up
- documentation on how to avoid these evictions

I'm not sure what the best approach would be here. Once it is clear I'm happy to attempt a PR.

**Logs**

Sometimes:

```
Reason:         Evicted
Message:        Pod The node had condition: [DiskPressure].
```

Mostly

```
Reason:         Evicted
Message:        The node was low on resource: ephemeral-storage. Container flux-helm-operator was using 8250536Ki, which exceeds its request of 0.
```

**Additional context**

- Helm Operator version: docker.io/fluxcd/helm-operator:1.0.1
- Kubernetes version: v1.14.9-eks-f459c0
- Git provider: github

**Comment by erkannt** [comment]
The cause of the disk usage is `/root/.cache/helm/repository` filled with thousands of

```
TEq5um2TBh56PLk4Gols5nOn3VU=-charts.txt  zx8k866dNOfxTYM9Ly8qMaUXZEA=-charts.txt
TEq5um2TBh56PLk4Gols5nOn3VU=-index.yaml  zx8k866dNOfxTYM9Ly8qMaUXZEA=-index.yaml
```

Is this common? Or is my setup fishy?

**Comment by hiddeco** [comment]
Updating to the latest version (`1.1.0`) of the operator will resolve this issue, as it has been fixed in #422.

**Comment by erkannt** [comment]
Thanks for pointing out the new release. Sorry for not checking that :see_no_evil: 

`helm repo update` followed by an `upgrade` has fixed the issue.
